devel/libmaa/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    libmaa
+version                 1.3.2
+categories              devel
+platforms               darwin
+license                 LGPL-2
+maintainers             {schenkel.net:leonardo @lbschenkel} openmaintainer
+
+description             Low-level data structures helpful for writing compilers
+long_description        ${description}
+homepage                https://sourceforge.net/projects/dict/
+
+master_sites            sourceforge:dict
```

Comment:
Please specify exact path.

```
--->  Checksumming libmaa-1.3.2.tar.gz
DEBUG: Calculated (rmd160) is 9d4c5d27cce9396191575146cde3e087d8d8946a
Error: Checksum (rmd160) mismatch for libmaa-1.3.2.tar.gz
Portfile checksum: libmaa-1.3.2.tar.gz rmd160 f3b091b8b5f8e8d7a69d192b25c70bf10cdf0d09
Distfile checksum: libmaa-1.3.2.tar.gz rmd160 9d4c5d27cce9396191575146cde3e087d8d8946a
DEBUG: Calculated (sha256) is 7a07d3f7cca5c0b38ca811984ef8da536da32932d68c1a6cce33ec2462b930bf
Error: Checksum (sha256) mismatch for libmaa-1.3.2.tar.gz
Portfile checksum: libmaa-1.3.2.tar.gz sha256 59a5a01e3a9036bd32160ec535d25b72e579824e391fea7079e9c40b0623b1c5
Distfile checksum: libmaa-1.3.2.tar.gz sha256 7a07d3f7cca5c0b38ca811984ef8da536da32932d68c1a6cce33ec2462b930bf
***
The non-matching file appears to be HTML. See this page for possible reasons
for the checksum mismatch:
<https://trac.macports.org/wiki/MisbehavingServers>
***
The file has been moved to: /opt/local/var/macports/distfiles/libmaa/libmaa-1.3.2.tar.gz.html
Error: Failed to checksum libmaa: Unable to verify file checksums
```

https://paste.macports.org/4ff298e615ac.txt

Reply:
That's very strange. I have just wiped all my distfiles and forced a redownload and I don't see this. Maybe due to different mirrors? In case I have to hardcode this, is there any preference to which mirror should be chosen?

Reply:
You should use `sourceforge:project/dict/libmaa/libmaa-${version}`.

Reply:
Changed. Still weird that it worked for me, though.